### PR TITLE
Moonlink.js supports lavalink v4

### PR DESCRIPTION
**Pull request by 1Lucas1apk**
Well, I had already left the base to work with v4, I was just waiting for a lavalink version to test and it's working correctly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,6 +90,7 @@ Version numbers can come in different combinations, depending on the release typ
 | Client                                             | Platform | Compatible With | Additional Information |
 |----------------------------------------------------|----------|-----------------|------------------------|
 | [DisGoLink](https://github.com/disgoorg/disgolink) | Go       | **Any**         |                        |
+| [Moonlink.js](https://github.com/1Lucas1apk/moonlink.js)    | Node.js  | **Any**                                    |                                 |
```
Comment (topi314) [inline README.md]: please format the table accordingly :)

**Review by topi314 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (1Lucas1apk) [reply README.md]: Now it is right?

Comment (topi314) [reply README.md]: not the slightest bit
image

Comment (1Lucas1apk) [reply README.md]: It's to leave the whole tab the same, right, I'm going to do that

Comment (1Lucas1apk) [reply README.md]: There you go, I think it's right now

Comment (topi314) [reply README.md]: image
there are still 2 random `|`

**Review by topi314 — APPROVED** [review]

**Pull request merged**